docs/release-notes/helm-requires-user-password.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**Security Fixes**
+
+-  Helm: When deploying a new cluster with Helm, an initial password for the ``admin`` and
+      ``determined`` users should be specified using either ``initialUserPassword`` or
+      ``defaultPassword`` (see helm/charts/determined/values.yaml). Configuring a password is no
+      longer done as a separate step.
```

Comment:
:orphan:

**Security**

-  Helm: When deploying a new cluster with Helm, configuring an initial password fofr the "admin" and "determined" users is required and is no longer a separate step. To specify an initial password for the ``admin`` and
      ``determined`` users, visit the ``helm/charts/determined/values.yaml`` file and use either ``initialUserPassword`` (preferred) or
      ``defaultPassword`` (deprecated). For reference, visit :ref:`helm-config-reference`.

Reply:
Thanks!